app/models/shopify_data_feed.rb

```diff
@@ -0,0 +1,5 @@
+class ShopifyDataFeed < ActiveRecord::Base
```

Comment:
@vfonic I'd like have it separated from Shopify namespace where will be ActiveRecord/ShopifyAPI objects.